### PR TITLE
[12.0 ]stock_request product quantity from virtual_available if negative …

### PR DESCRIPTION
**Pull request by JuanjoA-PH**
…value and not 1.0
PR to cover this: [issue/question](https://github.com/OCA/stock-logistics-warehouse/issues/1109)

### CODE REVIEW

**Comment by JuanjoA-PH** [comment]
Edit:
Hi, I apologize about the language issue. I edit to fix it.

Travis gives an error when passing the checks of this PR, but I don't know what the problem might be.

@pedrobaeza, sorry, can you help me with this?

Maybe this improvement may not be of interest to everyone, but I would like to have at least all the checks pass.

Thanks!

**Comment by pedrobaeza** [comment]
In repositories different from OCA/l10n-spain, you have to talk in English. Rebase your PR to see if it's a temporary glitch.

**Comment by JuanjoA-PH** [comment]
Hi @pedrobaeza , now the test are ok. Who can see if this is interesting to integrate in OCA?
Thanks
JuanjoA

**Comment by pedrobaeza** [comment]
Please squash commits. Maybe original authors can be interested not being a very old version: @HviorForgeFlow and @LoisRForgeFlow

**Comment by JuanjoA-PH** [comment]
@HviorForgeFlow , @LoisRForgeFlow can you check this PR to enable merge?

File: stock_request/models/stock_request_order.py [hunk 1]
```diff
@@ -301,7 +301,8 @@ def _create_from_product_multiselect(self, products):
                 stock_request_ids=[(0, 0, dict(
                     product_id=product.id,
                     product_uom_id=product.uom_id.id,
-                    product_uom_qty=1.0,
+                    product_uom_qty=1.0 if product.virtual_available >= 0
+                    else abs(product.virtual_available),
```
Comment (rousseldenis) [inline stock_request/models/stock_request_order.py:305]: Why virtual_available ? @JuanjoA-PH What's the aim of this ?

**Comment by rousseldenis** [comment]
@JuanjoA-PH

**Pull request closed without merging**